### PR TITLE
Add base instructions for the Scala image definition

### PR DESCRIPTION
**Pull request by hectcastro**
This is an attempt to produce a Docker container that includes Scala and `sbt`. It uses the official Java base container provided by Docker.

[Review on Reviewable](https://reviewable.io/reviews/azavea/docker-scala/1)

### CODE REVIEW

**Comment by sharph** [comment]
Reviewed 3 of 3 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---

<sup>**[Dockerfile, line 10 \[r1\]](https://reviewable.io:443/reviews/azavea/docker-scala/1#-JvK40qMMU0Pwba88KpK)** ([raw file](https://github.com/azavea/docker-scala/blob/b4d8c45aed81ecdab3aa4a1056d343db03e78783/Dockerfile#L10)):</sup>
Do you think it is worth consolidating these two `RUN` lines like you did with several commands below? I've been wondering if there is a best practice for this, or if we should adopt one.

On one hand, the overhead of having `RUN` lines that create a filesystem layer like `mkdir -p some_dir` is very low and really makes no difference in the functionality of the solution. On the other hand, having a layer consisting of one empty directory *feels* slightly less tidy.

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/azavea/docker-scala/1)

**Comment by hectcastro** [comment]
<sup>**[Dockerfile, line 10 \[r1\]](https://reviewable.io:443/reviews/azavea/docker-scala/1#-JvK40qMMU0Pwba88KpK)** ([raw file](https://github.com/azavea/docker-scala/blob/b4d8c45aed81ecdab3aa4a1056d343db03e78783/Dockerfile#L10)):</sup>
Right now I'm following techniques seen in "official" Docker [image definitions](https://github.com/docker-library/python/blob/7663560df7547e69d13b1b548675502f4e0917d1/2.7/onbuild/Dockerfile#L3). There are some [best practice collections](https://docs.docker.com/articles/dockerfile_best-practices/), but they can be hard to follow because many times the commands before and after play a factor in the approach.

I'm not opposed to combining now and going back through to fix if we find evidence that suggests we shouldn't.

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/azavea/docker-scala/1)

**Comment by sharph** [comment]
Reviewed 1 of 1 files at r2.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/azavea/docker-scala/1)
